package.json

```diff
@@ -24,28 +24,21 @@
   "license": "BSD-3-Clause",
   "main": "src/embed.js",
   "devDependencies": {
-    "browserify": "^13.0.0",
-    "browserify-shim": "^3.8.12",
-    "chai": "^3.5.0",
-    "d3": "^3.5.16",
-    "d3-geo-projection": "^0.2.16",
-    "jshint": "^2.9.1",
-    "mocha": "^2.4.5",
-    "topojson": "^1.6.24",
-    "tv4": "^1.2.7",
-    "uglify-js": "^2.6.2",
-    "vega": "^2.5.0",
-    "vega-lite": "^1.0.7",
-    "watchify": "^3.7.0"
+    "browserify": "^14.0.0",
+    "browserify-shim": "^3.8.13",
+    "eslint": "^3.14.1",
+    "uglify-js": "^2.7.5",
+    "vega": "3.0.0-beta.20",
+    "vega-lite": "2.0.0-alpha.0",
+    "watchify": "^3.9.0",
+    "vega-schema-url-parser": "^1.0.0-beta.1"
   },
```

Comment:
Probably need to depend on vega-schema-url-parser and d3-selection so we can compile it into our lib.